File: kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/ru/tinkoff/kora/example/submodule/vet/model/dao/Vet.java
package ru.tinkoff.kora.example.submodule.vet.model.dao;

import ru.tinkoff.kora.database.common.annotation.Column;
import ru.tinkoff.kora.database.common.annotation.Id;
import ru.tinkoff.kora.database.common.annotation.Table;

@Table("vets")
public record Vet(@Column("id") @Id long id,
                  @Column("name") String name,
                  @Column("name") String surname) {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/ru/tinkoff/kora/kotlin/example/submodule/vet/model/dao/Vet.kt
package ru.tinkoff.kora.kotlin.example.submodule.vet.model.dao

import ru.tinkoff.kora.database.common.annotation.Column
import ru.tinkoff.kora.database.common.annotation.Id
import ru.tinkoff.kora.database.common.annotation.Table
import ru.tinkoff.kora.database.jdbc.EntityJdbc

@EntityJdbc
@Table("vets")
data class Vet(
    @field:Column("id") @field:Id val id: Long,
    @field:Column("name") val name: String,
    @field:Column("surname") val surname: String,
)
